services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/download/DownloadListManagerImpl.java

```diff
@@ -460,16 +466,21 @@ AddToDownloadListResponse addQueryResultsToDownloadList(final ProgressCallback p
 	 */
 	DownloadListItem createDownloadsListItemFromRow(final boolean useVersion, Row row) {
```

Comment:
useVersion->inlucdeVersion
can we unconditionally use the values for both rowId and rowVersion?